MatrixKit/Controllers/MXKAttachmentsViewController.m

```diff
@@ -1044,6 +1078,12 @@ - (void)collectionView:(UICollectionView *)collectionView didEndDisplayingCell:(
             }
         }
     }
+  
+    //tell the delegate that a new Attachment has been shown and pass eventId
```

Comment:
I don't think this code is at the right place.
Indeed the current cell is not displayed anymore here.
Maybe you should use collectionView:willDisplayCell: method.